src/layouts/CreateProjectLayout.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
+import Grid from "@material-ui/core/Grid";
+import styled from "styled-components";
+
+import ProgressBar from "../../src/components/general/ProgressBar.js";
```

Comment:
It looks like we're backing out to the project root before coming back in. Could we shorten this import?

```jsx
 import ProgressBar from "../components/general/ProgressBar"; 
```

This comment applies to a few other imports in this PR, too.